src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -4192,4 +4192,29 @@ public static void updatePinsIsRouted(Design design) {
             updatePinsIsRouted(net);
         }
     }
+
+    /**
+     * Removes all the existing encrypted cell files from a design and replaces them
+     * with the provided lists and black boxes those cells. The provided files
+     * should be named after the cell type. This is useful in the scenarios where a
+     * design has many thousand of individual encrypted cell files that are time
+     * consuming to load. By providing a higher level of hierarchy cell definition,
+     * the number of individual loads can be reduced.
+     * 
+     * @param design   The design to modify.
+     * @param netlists The list of encrypted cell files (*.edn, *.edf, or *.dcp)
```

Comment:
I didn't have any success with Verilog files in the few attempts I made.

Reply:
I mentioned Verilog because you added a `read_verilog` to `EDIFTools.writeTclLoadScriptForPartialEncryptedDesigns()` which could imply support. Does it make sense to keep that too?

Reply:
I believe Verilog should be a valid path, I just didn't have success.